searx/engines/apkmirror.py

```diff
@@ -20,47 +28,41 @@
 # engine dependent config
 categories = ['it']
```

Comment:
The F-Droid search engine is in category `files` .. should we move APKMirror also into category `files`?

Reply:
I agree it should be in the `files` category.